lms/templates/discovery/filter_bar.underscore

```diff
@@ -1,5 +1,4 @@
-<ul class="active-filters facet-list">
-</ul>
-<span>
+<div class="filters-inner">
+    <ul class="active-filters facet-list"></ul>
     <button id="clear-all-filters" class="clear-filters flt-right discovery-button"><%= gettext('CLEAR ALL') %></button>
```

Comment:
Nope, the UI strings for this haven't been reviewed yet. I can get to that this afternoon.